<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Depositwindow</class>
 <widget class="QDialog" name="Depositwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="show_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>список пополнений</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>список снятий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Depos_pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>расчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма вклада к концу срока</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>551</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_deposit_amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_period">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_percent">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_tax_rate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QLabel" name="accured_interest">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>accrued interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>tax amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_amount_end">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>deposit amount</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="payment_periodicity">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>в конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>капитализация процентов</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_period">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>дней</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
